src/main/java/com/weirddev/testme/intellij/template/context/MockitoMockBuilder.java

```diff
@@ -137,6 +137,27 @@ public boolean hasMockable(List<Field> fields) {
         return false;
     }
 
+    /**
+     * for class with only private constructor that can not mock, for example util classes only with static methods
+     * @param testedClass tested class
+     * @return true - if tested class has public constructors
+     */
+    public boolean canMockViaCtors(Type testedClass) {
```

Comment:
- logic can be simplified:
```java
public boolean canMockViaCtors(Type testedClass) {
    List<Method> constructorList = testedClass.findConstructors();
    return constructorList.isEmpty() || constructorList.stream().anyMatch(method -> !method.isPrivate());
}
```
- I find the method name - _canMockViaCtors_ - confusing with a use case where mocks are passed as tested class c'tor arguments instead of annotating with `@InjectMocks`. I prefer a name such as _hasAccessibleCtor_

Reply:
thanks， I modified it。